### ISSUE TITLE
Hip and Upper leg weight painting error after prepping for either VRChat VRM?

### ISSUE THREAD

**Issue by TrentWatt2020** [FlailingFog/KK-Blender-Porter-Pack#550]
Tl;Dr Sit poses or poses that require the leg and knee to be moved look absolutely mangled after running "Prep for target app".

Before running the "Prep for Target App" button, using the IK rig to pose the character in a way that is sitting looks fine. The upper leg doesn't look weird or shift in some odd way. However... Posing the character AFTER prep absolutely mangles the upper leg. It causes the thigh to do some weird deform shenanigans. If you follow the little purple/pink lines I drew, it becomes really obvious that something is off with the weight painting. I've tried changing vertex groups, hand weight painting it, shifting the bones, playing with the deform, MMD Bone mod.. Nothing I do reaaaaally fixes things. 

It becomes most noticeable in VRChat, where sitting in a chair looks absolutely scuffed. Unity/VRCSDK also reports some weird rotational issues with the hips and pelvis region. I skimmed the issues and didn't really see any one else reporting this problem exactly how I see it. I've ported like 12 models through KK to Blender to VRC/VRM and never let it bother me till now. It's been a problem for the last like 2 years that I've noticed it. See the included images for a better idea of what I'm talking about. 

If you enable Viewport Overlay > Vertex group weights before prepping a model you'll also find that the leg and knee bones don't appear to have group weights..? But they do after prep, which leads me to believe either CATS merging and redoing weights is the culprit, or something similar.

The model sitting before running "Prep". Note the straight lines.
![sitting-before-prep](https://github.com/FlailingFog/KK-Blender-Porter-Pack/assets/141441333/02cdde5b-d9f3-4a4c-9f77-54deb5ae7210)

After running "Prep".
![sitting-after-prep](https://github.com/FlailingFog/KK-Blender-Porter-Pack/assets/141441333/5468e7a4-3bae-44c4-8035-31f5825d7a15)

After thought: I typically port male models, I didn't notice it with the one or two female models I've done. Might be something to do with the skeletal structure for males too.
